spec.md

```diff
@@ -2103,6 +2229,47 @@ Condition | gRPC Code | Description | Recovery Behavior
 | Call not implemented | 12 UNIMPLEMENTED | NodeGetInfo call is not implemented by the plugin or disabled in the Plugin's current mode of operation. | Caller MUST NOT retry. Caller MAY call `ControllerGetCapabilities` or `NodeGetCapabilities` to discover Plugin capabilities. |
 
 
+#### `NodeExpandVolume`
+
+A Node Plugin MUST implement this RPC call if it has `EXPAND_VOLUME` node capability.
+This RPC call allows CO to expand volume on a node.
+
+`NodeExpandVolume` ONLY supports expansion of already node-published or node-staged volumes on the given `volume_path`.
```

Comment:
Maybe say that this call can only expand volumes with the `VolumeExpansion.OFFLINE` expansion capability?

Reply:
Did you mean the opposite? i.e - NodeExpandVolume will mostly/only work with online volumes? Regardless - I thought we got past this. We agreed not to, tie `NodeExpandVolume` with `ONLINE` or `OFFLINE` capability explicitly in the spec.

Reply:
Oops. Yes I mean ONLINE. Sorry about the confusion because of this mistake.

Reply:
I'm confused by this specific comment thread. Is this resolved, or not?

Reply:
This is related to the discussion above. If an OFFLINE expansion involves a `NodeExpandVolume` the this sentence is incorrect. If not, then my comment here doesn't make sense.

Reply:
After an offline discussion with @gnufied it seems reasonable to leave this as is. Please drop my comment.